ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_multi_interface.hpp

```diff
@@ -38,7 +38,7 @@ class RRBotSystemMultiInterfaceHardware
 : public hardware_interface::BaseInterface<hardware_interface::SystemInterface>
 {
 public:
-  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemMultiInterfaceHardware);
+  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemMultiInterfaceHardware)
```

Comment:
Revert this change because it fails on our liners.

```suggestion
  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemMultiInterfaceHardware);
```